paddle/fluid/operators/has_data_op.cc

```diff
@@ -0,0 +1,70 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#include "paddle/fluid/operators/has_data_op.h"
+#include <string>
+#include "paddle/fluid/framework/op_registry.h"
+
+namespace paddle {
+namespace operators {
+
+class HasDataOpMaker : public framework::OpProtoAndCheckerMaker {
+ public:
+  void Make() override {
+    // inputs and outputs stored in proto
+    AddInput("X", "(LoDTensor) the LoDTensor to check");
+    AddOutput("Out", "(LoDTensor) the ouput of has_data_op");
```

Comment:
ouput --> output
has_data_op --> HasDataOp
Please add a dot at end of each sentence.